jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -68,6 +70,31 @@
   public static final TestProject defaultTargetTestProject =
       new TestProject(testPlugin, "default-target");
 
+  @ClassRule
+  public static final TestProject servlet25Project = new TestProject(testPlugin, "war_servlet25");
+
+  private static String getGcrImageReference(String imageName) {
+    String repository = "gcr.io/" + IntegrationTestingConfiguration.getGCPProject() + '/';
```

Comment:
nit: this is actually the registry plus the repository

Reply:
I think "nameBase" will work.